src/dbnode/persist/fs/migration/migration_test.go

```diff
@@ -111,14 +121,27 @@ func TestToVersion1_1Run(t *testing.T) {
 	require.NotEqual(t, oldBytes, newBytes)
 
 	// Corrupt bytes to trip newly added checksum
-	decoder := msgpack.NewDecoder(nil)
 	newBytes[len(newBytes)-1] = 1 + newBytes[len(newBytes)-1]
 	decoder.Reset(msgpack.NewByteDecoderStream(newBytes))
 	_, err = decoder.DecodeIndexEntry(nil)
 	require.Error(t, err)
 	require.Contains(t, err.Error(), "checksum mismatch")
 }
 
+func openFile(
+	t *testing.T,
+	fsOpts fs.Options,
+	nsId ident.ID,
+	shard uint32,
+	infoFileResult fs.ReadInfoFileResult,
+	fileType string,
+) (*os.File, error) {
+	indexFd, err := os.Open(path.Join(fsOpts.FilePathPrefix(), fmt.Sprintf("data/%s/%d/fileset-%d-%d-%s.db",
+		nsId.String(), shard, infoFileResult.Info.BlockStart, infoFileResult.Info.VolumeIndex, fileType)))
+	require.NoError(t, err)
+	return indexFd, err
```

Comment:
nit: this doesn't need to return an error since failing a `require` will stop the current test

Reply:
Derp, just a bad refactor.